part-3/pk-sim-expression-profile.md

```diff
@@ -443,7 +442,7 @@ Activating/deactivating checkboxes in each of these 3 localization groups change
         - `Fraction expressed in endosomes ` is shown and can be edited by user
         - `Fraction expressed on plasma-side membrane of vascular endothelium ` is shown and can be edited by user
         - `Fraction expressed on tissue-side membrane of vascular endothelium ` is hidden and always set to `1 - (Fraction expressed in endosomes + Fraction expressed on plasma-side membrane of vascular endothelium)`
-![](../assets/images/part-3/FractionExpressed2.png)
+      ![](../assets/images/part-3/FractionExpressed2.png)
```

Comment:
not sure why this blanks were inserted, but when rendered with gitbook it looks not as nice as before.
Now:
![grafik](https://user-images.githubusercontent.com/25061876/159333122-48df43a6-b13d-49b4-a2f6-7dc458507220.png)

Before:
![grafik](https://user-images.githubusercontent.com/25061876/159333190-0d105845-e5ac-435f-baee-48852d6b5458.png)

Reply:
Have reverted this, hope its good now.